From 675757197d8e2f37082cfca4fbc7eda5da62537d Mon Sep 17 00:00:00 2001
From: Yongxin Liu <yongxin.liu@windriver.com>
Date: Wed, 20 Mar 2024 09:31:47 +0000
Subject: [PATCH] rdk/grr: klm/cpk/ice_devlink.c: remove devlink_set_features()

devlink_set_features() was removed in kernel commit fb8421a94c56
("devlink: remove devlink features").

Upstream-Status: Inappropriate [RDK is released in tarball only]

Signed-off-by: Yongxin Liu <yongxin.liu@windriver.com>
---
 klm/cpk/ice_devlink.c | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/klm/cpk/ice_devlink.c b/klm/cpk/ice_devlink.c
index 4f1b8e6..ccc1abe 100644
--- a/klm/cpk/ice_devlink.c
+++ b/klm/cpk/ice_devlink.c
@@ -1019,9 +1019,6 @@ void ice_devlink_register(struct ice_pf *pf)
 {
 	struct devlink *devlink = priv_to_devlink(pf);
 
-#ifdef HAVE_DEVLINK_SET_FEATURES
-	devlink_set_features(devlink, DEVLINK_F_RELOAD);
-#endif /* HAVE_DEVLINK_SET_FEATURES */
 #ifdef HAVE_DEVLINK_REGISTER_SETS_DEV
 	devlink_register(devlink, ice_pf_to_dev(pf));
 #else
-- 
2.39.3

